internal/statestore/redis.go

```diff
@@ -222,418 +213,3 @@ func redisURLFromAddr(addr string, cfg config.View, usePassword bool) string {
-func (rb *redisBackend) GetAssignments(ctx context.Context, id string, callback func(*pb.Assignment) error) error {
-	redisConn, err := rb.redisPool.GetContext(ctx)
-	if err != nil {
-		return status.Errorf(codes.Unavailable, "GetAssignments, id: %s, failed to connect to redis: %v", id, err)
-	}
-	defer handleConnectionClose(&redisConn)
-
-	backoffOperation := func() error {
-		var ticket *pb.Ticket
-		ticket, err = rb.GetTicket(ctx, id)
-		if err != nil {
-			return backoff.Permanent(err)
-		}
-
-		err = callback(ticket.GetAssignment())
-		if err != nil {
-			return backoff.Permanent(err)
-		}
-
-		return status.Error(codes.Unavailable, "listening on assignment updates, waiting for the next backoff")
-	}
-
-	err = backoff.Retry(backoffOperation, rb.newConstantBackoffStrategy())
-	if err != nil {
-		return err
-	}
-	return nil
-}
-
-// AddTicketsToPendingRelease appends new proposed tickets to the proposed sorted set with current timestamp
-func (rb *redisBackend) AddTicketsToPendingRelease(ctx context.Context, ids []string) error {
-	if len(ids) == 0 {
-		return nil
-	}
-
-	redisConn, err := rb.redisPool.GetContext(ctx)
-	if err != nil {
-		return status.Errorf(codes.Unavailable, "AddTicketsToPendingRelease, failed to connect to redis: %v", err)
-	}
-	defer handleConnectionClose(&redisConn)
-
-	currentTime := time.Now().UnixNano()
-	cmds := make([]interface{}, 0, 2*len(ids)+1)
-	cmds = append(cmds, proposedTicketIDs)
-	for _, id := range ids {
-		cmds = append(cmds, currentTime, id)
-	}
-
-	_, err = redisConn.Do("ZADD", cmds...)
-	if err != nil {
-		err = errors.Wrap(err, "failed to append proposed tickets to pending release")
-		return status.Error(codes.Internal, err.Error())
-	}
-
-	return nil
-}
-
-// DeleteTicketsFromPendingRelease deletes tickets from the proposed sorted set
-func (rb *redisBackend) DeleteTicketsFromPendingRelease(ctx context.Context, ids []string) error {
-	if len(ids) == 0 {
-		return nil
-	}
-
-	redisConn, err := rb.redisPool.GetContext(ctx)
-	if err != nil {
-		return status.Errorf(codes.Unavailable, "DeleteTicketsFromPendingRelease, failed to connect to redis: %v", err)
-	}
-	defer handleConnectionClose(&redisConn)
-
-	cmds := make([]interface{}, 0, len(ids)+1)
-	cmds = append(cmds, proposedTicketIDs)
-	for _, id := range ids {
-		cmds = append(cmds, id)
-	}
-
-	_, err = redisConn.Do("ZREM", cmds...)
-	if err != nil {
-		err = errors.Wrap(err, "failed to delete proposed tickets from pending release")
-		return status.Error(codes.Internal, err.Error())
-	}
-
-	return nil
-}
-
-func (rb *redisBackend) ReleaseAllTickets(ctx context.Context) error {
-	redisConn, err := rb.redisPool.GetContext(ctx)
-	if err != nil {
-		return status.Errorf(codes.Unavailable, "ReleaseAllTickets, failed to connect to redis: %v", err)
-	}
-	defer handleConnectionClose(&redisConn)
-
-	_, err = redisConn.Do("DEL", proposedTicketIDs)
-	return err
-}
-
-func handleConnectionClose(conn *redis.Conn) {
```

Comment:
This is used by backfill as well, should stay in redis.go?

Reply:
yes, missed that. thank you